packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -232,6 +232,16 @@ export interface FunctionOptions extends EventInvokeConfigOptions {
    */
   readonly logRetentionRole?: iam.IRole;
 
+  /**
+   * Retry options for creating CloudWatch log groups. Deploying many Lambdas
+   * with log retention resources may result in rate limit issues when creating
+   * CloudWatch Log groups. The retry options allow you to customize the retry
+   * options in order to successfully create these.
```

Comment:
Sufficient to just document what these options are. There can be many reasons to use them.

Something like this -

```suggestion
   * When log retention is specified, a custom resource attempts to create the CloudWatch log group.
   * These options control the retry policy when interacting with CloudWatch APIs.
```

Reply:
Agree, will add that.